BWT/BWT/BWTAlgorithm.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace BWT
+{
+    using System.Text;
+    public class BWTAlgorithm
+    {
+        public static (string, int) FrontBWT(string input)
```

Comment:
Типы кортежа по стайлгайду надо именовать, с большой буквы
```suggestion
        public static (string ..., int ...) FrontBWT(string input)
```